react-frontend/client/containers/App.js

```diff
@@ -1,7 +1,6 @@
 import React, { Component } from 'react';
 import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
-
```

Comment:
Do we have an `eslint` file we wanna use?

If we're going be styling picky, I'd like to be able to easily adhere in my code.

Reply:
Yes we can integrate the eslint but I would like to see 4 spaces for indents instead of 2 much much easier for my eyes. Yes we can do that for imports.